ch.imatt.ClipQR.yml

```diff
@@ -0,0 +1,42 @@
+app-id: ch.imatt.ClipQR
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+command: clipqr
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+
+build-options:
+  build-args: [ "--share=network" ]
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+  - name: clipqr
+    buildsystem: simple
+    build-commands:
+      - $GOROOT/bin/go build -o clipqr
+      - install -Dm00755 clipqr $FLATPAK_DEST/bin/clipqr
+      - >
+        for size in 16 24 32 48 64 128 256 512; do
+          rsvg-convert -w $size -h $size -f png -o $size.png icon.svg
+          install -Dm644 $size.png /app/share/icons/hicolor/${size}x${size}/apps/$FLTPAK_ID.png
+        done
+      - install -Dm00644 icon.svg $FLATPAK_DEST/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -Dm00644 $FLATPAK_ID.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -Dm00644 $FLATPAK_ID.appdata.xml $FLATPAK_DEST/share/appdata/$FLATPAK_ID.appdata.xml
+    sources:
+      - type: archive
+        url: https://gitlab.com/imatt-foss/clipqr/-/archive/v1.0.0/clipqr-v1.0.0.tar.gz
+        sha256: 1eb9a6ee7e431c252e30a087d0c72868c27931a62ff3219f597d78c7f3b45d7f
+      - type: file
+        path: ch.imatt.ClipQR.appdata.xml
+      - type: file
+        path: ch.imatt.ClipQR.desktop
```

Comment:
have the .desktop and appdata been submitted upstream?

A Linux desktop app without a `.desktop` is a bug.

Reply:
Nope but I will commit them upstream, thanks again